### DISCUSSION TITLE
about obj2sdf, I use obj2sdf to split threecubes, however I get some error when I run threecubes.py

### DISCUSSION THREAD

**Discussion by WJ1214** [bulletphysics/bullet3#4120] [General]
I noticed that if model has more than one part, we need to transform .obj files to .sdf files. And I have recompiled bullet3 as in [1934](https://github.com/bulletphysics/bullet3/issues/1934), however, after I run `../../bin/App_obj2sdf_* --fileName="path-to-threecubes.obj"`, then some error raise up when I run python threecubes.py. Erros like this:
![image](https://user-images.githubusercontent.com/51085092/147474544-04ba91d1-2b1a-47d4-b611-1cce038b9208.png)
Did I do something wrong? How can I solve this?

**Comment by WJ1214** [comment]
I compared the generated SDF file with the original file and found that the generated file is different from the original file of the project. Is there a problem with the obj2sdf program?

**Comment by Akshay6077** [comment]
Hey did you figure out the problem I am facing the same issue?
